index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="style.css" />
+		<title>Document</title>
+	</head>
+	<body>
+		<div class="container"></div>
+
+		<div class="modal">
+			<form class="book-form">
+        <label for="title">Title</label>
+        <input class="book-title" name="title" type="text" />
+        <label for="author">Author</label>
+        <input class="book-author" name="author" type="text" />
+        <label for="pages">Pages</label>
+        <input class="book-pages" name="pages" type="number" />
```

Comment:
[Optional]
Usually, a book does not have any negative page numbers. So allowing a user to do that is against the default convention. So you can control this scenario, by updating the input field in the following way,
```html
<input class="book-pages" name="pages" type="number" min="0"  />
```